azure-pipelines.yml

```diff
@@ -1,3 +1,4 @@
+name: $(GitVersion.NuGetVersionV2)
```

Comment:
I believe this is incorrect - check the Running Inside TFS section of the GitVersion docs https://gitversion.net/docs/build-server-support/build-server/azure-devops for the proper format when using in the pipeline name